Atomic/rpm_host_install.py

```diff
@@ -12,16 +13,20 @@ class RPMHostInstall(object):
     @staticmethod
     def copyfile(src, dest):
         if os.path.isdir(src):
-            # add the directory only if it is empty, so we don't create directories that
+            # add the directory only if it is empty, so we don't delete directories that
```

Comment:
**Q**, hmm, I don't quite get ... how "delete" got involved in this function, can you explain it a bit :).

Reply:
we keep track of what files/directories we are adding.  We add the directory only if it is not already present.
In this way when we uninstall the container we will not delete directories that were already present when we first installed the container